tests/tox.ini

```diff
@@ -15,19 +15,21 @@ pass_env =
 description = Apply coding style standards to code
 deps = -r {tox_root}/requirements-dev.txt
 commands =
-    licenseheaders -t {tox_root}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {tox_root}/tests
-    isort {tox_root}/tests --profile=black
-    black {tox_root}/tests
+    licenseheaders -t {tox_root}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {tox_root}/sanity
+    licenseheaders -t {tox_root}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {tox_root}/integration
+    isort {tox_root}/sanity {tox_root}/integration --profile=black
+    black {tox_root}/sanity {tox_root}/integration
 
 [testenv:lint]
 description = Check code against coding style standards
 deps = -r {tox_root}/requirements-dev.txt
 commands =
-    codespell {tox_root}/tests
-    flake8 {tox_root}/tests
-    licenseheaders -t {tox_root}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {tox_root}/tests --dry
-    isort {tox_root}/tests --profile=black --check
-    black {tox_root}/tests --check --diff
+    codespell {tox_root}/sanity {tox_root}/integration
+    flake8 {tox_root}/sanity {tox_root}/integration
+    licenseheaders -t {tox_root}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {tox_root}/sanity --dry
+        licenseheaders -t {tox_root}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {tox_root}/integration --dry
+    isort {tox_root}/sanity {tox_root}/integration --profile=black --check
+    black {tox_root}/sanity {tox_root}/integration --check --diff
```

Comment:
we should run codespell, flake8, licenceheaders, isort, and black on both ``integration`` and ``sanity`` folders.

Reply:
I believe this should be already resolved, have I missed something?

Reply:
From what I see here, only ``licenseheaders`` is ran on both ``integration`` and ``sanity`` folders. The others are only being run on ``integration``. Same on the ``format`` target.

Reply:
Nevermind, I'm blind, both ``sanity`` and ``integration`` are there. :) LGTM.